dist-git/run_tests.sh

```
@@ -6,6 +6,16 @@ set -e
 common_path=$(readlink -f ../common)
 export PYTHONPATH="${PYTHONPATH+$PYTHONPATH:}$common_path"
 
+REDIS_PORT=7777
+redis-server --port $REDIS_PORT &> _redis.log &
```

Shellcheck (0.11.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)